src/kava.js

```diff
@@ -114,7 +114,9 @@ class Kava extends BasePlugin {
   _reset(): void {
     this._timer.destroy();
     this._rateHandler.destroy();
-    this._performanceObserver.disconnect();
+    if (this._performanceObserver && typeof this._performanceObserver.disconnet === 'function') {
```

Comment:
why is the typeof check required? I don't see a case where PerformanceObserver is defined but instantiating it will yield an instance that doesn't have the `disconnet` method.
And in any case there a typo in the check so it will never cal disconnect and will cause memory leaks.
```suggestion
    if (this._performanceObserver && typeof this._performanceObserver.disconnect === 'function') {
```
But before fixing it please check my comment above

Reply:
BTW, you mention in title that Kaltura player tests fail? doesn't the kava tests fail?

Reply:
Kava success as you can see here in travis status